src/engine/Value.v3

```diff
@@ -199,4 +199,14 @@ component Values {
 		}
 		return (null, -1);
 	}
+	def kind(val: Value) -> ValueKind {
```

Comment:
the `toTag` method in spc takes a `ValueType` instead of a `Value` as an argument, so I still need to keep this method. However, there's another `ValueTypes.kind` function that does the same thing, I'll update spc to use that function